ScoutSuite/providers/azure/resources/storageaccounts/storageaccounts.py

```diff
@@ -0,0 +1,55 @@
+import asyncio
+
+from ScoutSuite.providers.azure.resources.resources import AzureCompositeResources
+from ScoutSuite.providers.azure.utils import get_resource_group_name
+from ScoutSuite.providers.utils import get_non_provider_id
+from ScoutSuite.providers.azure.facade.storageaccounts import StorageAccountsFacade
+
+from .blob_containers import BlobContainers
+
+
+class StorageAccounts(AzureCompositeResources):
+    _children = [
+        (BlobContainers, 'blob_containers')
+    ]
+
+    async def fetch_all(self, credentials, **kwargs):
+        # TODO: build that facade somewhere else:
+        facade = StorageAccountsFacade(credentials.credentials, credentials.subscription_id)
+
+        self['storage_accounts'] = {}
+        for raw_storage_account in await facade.get_storage_accounts():
```

Comment:
I feel like having this kind of breaks the convention. It's the same as if you had a child named `storage_accounts`, basically.

Reply:
Ok, I see why you did this. Still breaks the structure. Is there a way we can reorganize this so we don't break the conventions?

Reply:
It's not the only place where I'm doing that unfortunately... It's strictly for compatibility reasons and I was planning a PR to get rid of that as soon as we will be moving to the new architecture.

Reply:
Sounds good!